lib/GoogleTest/GoogleTestFinder.cpp

```diff
@@ -228,19 +238,6 @@ std::vector<std::unique_ptr<Test>> GoogleTestFinder::findTests(Context &Ctx) {
   return tests;
 }
 
-std::string demangle(const char* name) {
```

Comment:
👍 but we might need it for a display name later on in some other class (to display logs for example?).